### ISSUE TITLE
HI my question

### ISSUE THREAD

**Issue by ayuan04111** [jsmartini/Bitcoin_LSTM#1]
![image](https://user-images.githubusercontent.com/84262965/118387993-3b1a0080-b654-11eb-90c1-436ce76e803a.png)

EPOCH 0/20
E:\Anaconda\envs\python36\lib\site-packages\torch\nn\modules\loss.py:528: UserWarning: Using a target size (torch.Size([])) that is different to the input size (torch.Size([1, 1, 1])). This will likely lead to incorrect results due to broadcasting. Please ensure they have the same size.
  return F.mse_loss(input, target, reduction=self.reduction)
